code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/internal/MessagingExtension.java

```diff
@@ -77,7 +77,7 @@
 public final class MessagingExtension extends Extension {
     private final static String SELF_TAG = "MessagingExtension";
 
-    final InAppNotificationHandler inAppNotificationHandler;
+    final AJOPayloadHandler AJOPayloadHandler;
```

Comment:
nit: Should we rename `AJOPayloadHandler` to `EdgePersonalizationResponseHandler` to keep it upstream solution agnostic.

Reply:
sounds good, will update the name